src/introduction/basic-concepts.md

```diff
@@ -179,7 +179,7 @@ vendors.
 
 - Semantic Versioning - <https://semver.org/>
 - The CWL Specification page in the CWL website: <https://www.commonwl.org/specification/>
-    - [See the Command Line Tool Description Standard](https://www.commonwl.org/v1.2/CommandLineTool.html)
+- See the Command Line Tool Description Standard: <https://www.commonwl.org/v1.2/CommandLineTool.html>
```

Comment:
The rest of the entries don't have "See"  I would remove to make it uniform with the rest

Reply:
'The command line Tool Description Standard' is this preferred